src/test/common/page/choose-channel-for-code.js

```diff
@@ -37,6 +37,10 @@ class ChooseChannelForCode extends SubmittablePageWithRadioButtons {
   async selectTextRadioButton () {
     await super.selectRadioButton('text')
   }
+
+  async selectEmailRadioButton () {
+    await super.selectRadioButton('email')
```

Comment:
I think `email` and `text` should be constants. I was going to say that on the previous PR but you merged it before I could review it!

Reply:
Constants added.